dist/angular-ui-tree.js

```diff
@@ -1380,7 +1422,7 @@
               angular.element($document).bind('mouseup', dragEndEvent);
               angular.element($document).bind('mousemove', dragMoveEvent);
               angular.element($document).bind('mouseleave', dragCancelEvent);
-              angular.element($document).bind('keydown', keydownHandler);
+              angular.element($document).bind('keyup', keyupHandler);
```

Comment:
could you add a comment why the key up handler is needed vs the key down, just in case we try to switch it back to key down in the future